extensions/cesium-renderer/src/decorations/PrimitiveConverter.ts

```diff
@@ -147,27 +147,33 @@ export abstract class PrimitiveConverter<TPrimitiveData = DecorationPrimitiveEnt
   }
 
 
-  /** Convert all decoration types using switch-based auto dispatch */
-  public convertAllDecorationTypes(decorations: Decorations, scene: CesiumScene, iModel?: IModelConnection): void {
-    if (decorations.world) {
-      this.autoDispatchGraphics(decorations.world, 'world', scene, iModel);
-    }
+  /** Convert decorations of a specific type, optionally clearing existing Cesium primitives for that type first. */
```

Comment:
Are the substantial changes to a number of decoration converter classes intentional?